src/graph/util/ExpressionUtils.cpp

```diff
@@ -189,6 +189,24 @@ Expression *ExpressionUtils::rewriteAgg2VarProp(const Expression *expr) {
   return RewriteVisitor::transform(expr, std::move(matcher), std::move(rewriter));
 }
 
+Expression *ExpressionUtils::rewriteSubExprs2VarProp(const Expression *expr,
+                                                     std::vector<Expression *> &subExprs) {
+  ObjectPool *pool = expr->getObjPool();
+  auto matcher = [&subExprs](const Expression *e) -> bool {
+    for (auto subExpr : subExprs) {
+      if (e->toString() == subExpr->toString()) {
+        return true;
+      }
+    }
+    return false;
+  };
```

Comment:
rewriteSubExpr2VarProp(const Expresssion* expr, Expression* subExpr)

Reply:
?

Reply:
I need to rewrite some subExprs, not one.

Reply:
ACK.